tests/e2e/actions.go

```diff
@@ -1847,8 +1848,12 @@ func (tr TestRun) assignConsumerPubKey(action assignConsumerPubKeyAction, verbos
 	}
 
 	if action.expectError {
+		if err == nil {
```

Comment:
```suggestion
		if err == nil || !strings.Contains(string(bz), "a validator has assigned the consumer key already: consumer key is already in use by a validator") {
```

Just looking at the PR, my first instinct was that the error we are catching after putting `gas auto` just relates somehow to the gas change. What do you think about putting something like this here to make sure we don't get back an unrelated error?

Reply:
good point. changed a bit the proposal to be more generic so that the expected error is provided by the step expecting the failure.

Reply:
Thanks, sounds even better